### ISSUE TITLE
can't figure out how to open a file with this

### ISSUE THREAD

**Issue by hemna** [trevarj/tmux-open-nvim#6]
I installed the plugin with tpm, but I don't understand how to open a file.

I symlinked ton into my PATH, but it seems to complain when I run it.
I am running macos and zsh as my shell.

```
╭─aprsd on  master [$?] via 🐍 v3.10.13 ❯
└─> ton time.py
/Users/I530566/.local/bin/ton: line 2: shopt: globstar: invalid shell option name
/Users/I530566/.local/bin/ton: line 6: /Users/I530566/.local/bin/variables.sh: No such file or directory
/Users/I530566/.local/bin/ton: line 7: /Users/I530566/.local/bin/helpers.sh: No such file or directory
/Users/I530566/.local/bin/ton: line 9: get_tmux_option: command not found
/Users/I530566/.local/bin/ton: line 10: get_tmux_option: command not found
/Users/I530566/.local/bin/ton: line 11: get_tmux_option: command not found
/Users/I530566/.local/bin/ton: line 12: get_tmux_option: command not found
/Users/I530566/.local/bin/ton: line 33: readarray: command not found
You must supply a filename. Found
```

**Comment by trevarj** [comment]
Hi @hemna, thanks for reporting. 

Well, this is embarrassing because I knowingly broke support for MacOS by using newer bash functions, such as `readarray`. And then broke the symlink capability by needing to source variables.sh and helpers.sh.

**A workaround**, until I can make the script compatible with the old bash that is shipped with MacOS (Bash 3.2.57) is to upgrade to a later bash using `brew install bash`

I have fixed the symlink issue in [34da7ca](https://github.com/trevarj/tmux-open-nvim/commit/34da7ca5c1cff5c696092ecdab9d8c98f1864633), so your symlink should work after upgrading bash.

Another way to use the script, other than on the command line, is to use something like [tmux-fingers](https://github.com/Morantron/tmux-fingers), which can execute the script for you. See the setup [here](https://github.com/trevarj/tmux-open-nvim?tab=readme-ov-file#tmux-fingers-or-tmux-open).

**Comment by trevarj** [comment]
Closing since the upgrade of bash should be a workaround until I can get to doing #5 or revert to old bash support.
